src/Azos/Apps/CommonApplicationLogic.Init.cs

```diff
@@ -19,66 +18,63 @@
 using Azos.Security;
 using Azos.Time;
 
-namespace Azos.Apps { partial class CommonApplicationLogic {
-
-
+namespace Azos.Apps
+{
+  partial class CommonApplicationLogic
+  {
     protected virtual void InitApplication()
     {
       if (ForceInvariantCulture)//used in all server applications
         Platform.Abstraction.PlatformAbstractionLayer.SetProcessInvariantCulture();
 
-
       var exceptions = new List<Exception>();
 
       var starters = GetStarters().ToList();
 
       string name = CoreConsts.UNKNOWN;
       bool breakOnError = true;
       foreach (var starter in starters)
-          try
-          {
-              breakOnError = starter.ApplicationStartBreakOnException;
-              name = starter.Name ?? starter.GetType().FullName;
-              starter.ApplicationStartBeforeInit(this);
-          }
-          catch (Exception error)
-          {
-              error = new AzosException(StringConsts.APP_STARTER_BEFORE_ERROR.Args(name, error.ToMessageWithType()), error);
-              if (breakOnError) throw error;
-              exceptions.Add(error);
-              //log not available at this point
-          }
-
+        try
+        {
+          breakOnError = starter.ApplicationStartBreakOnException;
+          name = starter.Name ?? starter.GetType().FullName;
+          starter.ApplicationStartBeforeInit(this);
+        }
+        catch (Exception error)
+        {
+          error = new AzosException(StringConsts.APP_STARTER_BEFORE_ERROR.Args(name, error.ToMessageWithType()), error);
+          if (breakOnError) throw error;
+          exceptions.Add(error);
+          //log not available at this point
+        }
 
       ExecutionContext.__BindApplication(this);
       DoInitApplication(); //<----------------------------------------------
       DoModuleAfterInitApplication();
 
-
       name = CoreConsts.UNKNOWN;
       breakOnError = true;
-      foreach(var starter in starters)
```

Comment:
If you remove indent under for each then braces need to be added to signify that try is under the loop, otherwise the code is unclear